packages/vite-plugin-cloudflare/src/node-js-compat.ts

```diff
@@ -43,21 +41,38 @@ export function isNodeCompat(
 }
 
 /**
- * If the current environment needs Node.js compatibility,
- * then inject the necessary global polyfills into the code.
+ * Get a set of module specifiers for all possible Node.js compat polyfill entry-points
+ */
+export function getNodeCompatEntries() {
+	const entries = new Set<string>(Object.values(env.alias));
+	for (const globInject of Object.values(env.inject)) {
+		if (typeof globInject === "string") {
+			entries.add(globInject);
+		} else {
+			entries.add(globInject[0] as string);
```

Comment:
what about
```suggestion
			entries.add(globInject[0]!);
```
?

Reply:
We actively have linting rules against use of !, so I would have to also add a lint comment to turn off that rule.
I'll fix it.

Reply:
4de9f48d9c6af8eb2319b4cb8ce5f4f892b99c53

Reply:
> We actively have linting rules against use of !, so I would have to also add a lint comment to turn off that rule.

yeah I was expecting that, but I did try in vscode to add `!` and vscode didn't seem to complain about it 🤷‍♂️  (I did not try to run the linting script though)